swift_browser_ui_frontend/src/views/AddContainer.vue

```diff
@@ -12,8 +12,8 @@
     </h1>
     <b-field
       horizontal
-      :label="$t('message.container_ops.containerName')"
-      :message="$t('message.container_ops.containerMessage')"
```

Comment:
Oops, this shouldn't have been removed.

Reply:
then we need to bring it back before merging

Reply:
fixed